app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::TopicsController < ApplicationController
+  # con este skip me salteo el token, después se vuelve a activar?
+  skip_before_action :verify_authenticity_token
+  
+  def index
+    # head :ok
+    topics = Topic.all
+    render json: topics, status: 200
+  end
+
+  def show
+  end
+
+  def create
+    topic = Topic.new(
+      label: topic_params[:label],
+      icon: topic_params[:icon]
```

Comment:
you dont need to set again `label` and `icon` attributes here, if you just use `Topic.new(topic_params)`, it will look exactly like this already